src/plugins/eqht-css/reports/1-acr-wcag21.json

```diff
@@ -0,0 +1,494 @@
+      "dct:modified": "2023-03-02",
+      "earl:mode": "earl:manual",
+      "@type": [ "acr:ConformanceRequirement" ]
+    },
+    {
+      "acr:requirement": "WCAG21:no-keyboard-trap",
+      "acr:conformance": "acr:satisfied",
+      "acr:assessment": [
+        "https://wet-boew.github.io/wet-boew/demos/acr/demo-wcag21/eqht-css-assessment-wcag21.json"
+      ],
+      "earl:subject": "_:subject",
+      "dct:description": "",
+      "dct:modified": "2023-03-02",
+      "earl:mode": "earl:manual",
+      "@type": [ "acr:ConformanceRequirement" ]
+    },
+    {
+      "acr:requirement": "WCAG21:character-key-shortcuts",
+      "acr:conformance": "acr:satisfied",
+      "acr:assessment": [
+        "https://wet-boew.github.io/wet-boew/demos/acr/demo-wcag21/eqht-css-assessment-wcag21.json"
+      ],
+      "earl:subject": "_:subject",
+      "dct:description": "",
+      "dct:modified": "2023-03-02",
+      "earl:mode": "earl:manual",
+      "@type": [ "acr:ConformanceRequirement" ]
+    },
+    {
+      "acr:requirement": "WCAG21:timing-adjustable",
+      "acr:conformance": "acr:satisfied",
+      "acr:assessment": [
+        "https://wet-boew.github.io/wet-boew/demos/acr/demo-wcag21/eqht-css-assessment-wcag21.json"
+      ],
+      "earl:subject": "_:subject",
+      "dct:description": "",
+      "dct:modified": "2023-03-02",
+      "earl:mode": "earl:manual",
+      "@type": [ "acr:ConformanceRequirement" ]
+    },
+    {
+      "acr:requirement": "WCAG21:pause-stop-hide",
+      "acr:conformance": "acr:satisfied",
+      "acr:assessment": [
+        "https://wet-boew.github.io/wet-boew/demos/acr/demo-wcag21/eqht-css-assessment-wcag21.json"
+      ],
+      "earl:subject": "_:subject",
+      "dct:description": "",
+      "dct:modified": "2023-03-02",
+      "earl:mode": "earl:manual",
+      "@type": [ "acr:ConformanceRequirement" ]
+    },
+    {
+      "acr:requirement": "WCAG21:three-flashes-or-below-threshold",
+      "acr:conformance": "acr:satisfied",
+      "acr:assessment": [
+        "https://wet-boew.github.io/wet-boew/demos/acr/demo-wcag21/eqht-css-assessment-wcag21.json"
+      ],
+      "earl:subject": "_:subject",
+      "dct:description": "",
+      "dct:modified": "2023-03-02",
+      "earl:mode": "earl:manual",
+      "@type": [ "acr:ConformanceRequirement" ]
+    },
+    {
+      "acr:requirement": "WCAG21:byNot tested-blocks",
+      "acr:conformance": "acr:satisfied",
+      "acr:assessment": [
+        "https://wet-boew.github.io/wet-boew/demos/acr/demo-wcag21/eqht-css-assessment-wcag21.json"
+      ],
+      "earl:subject": "_:subject",
+      "dct:description": "",
+      "dct:modified": "2023-03-02",
+      "earl:mode": "earl:manual",
+      "@type": [ "acr:ConformanceRequirement" ]
+    },
+    {
+      "acr:requirement": "WCAG21:page-titled",
+      "acr:conformance": "acr:satisfied",
+      "acr:assessment": [
+        "https://wet-boew.github.io/wet-boew/demos/acr/demo-wcag21/eqht-css-assessment-wcag21.json"
```

Comment:
Fix the assessment URL and all other instance. See the note in the report assessment summary.